### ISSUE TITLE
Why the 3D estimations obtained by GT 2D keypoints are the worst?

### ISSUE THREAD

**Issue by luzzou** [juyongchang/PoseLifter#1]
Hi, thank you for releasing such a wonderful work!
I have doubts about the results in Table 3, why the use of ground truth 2D pose provides
the worst results? It looks strange.

Looking forward to your reply!

**Comment by juyongchang** [comment]
We obtained the numbers in Table 3 using the 2D poses generated by the 2D pose estimator in Section 4.1 as a test set. Due to the estimation error, this test set has a statistically different distribution from the training set composed of GT 2D poses. Therefore, this case shows the worst result compared to other training sets (i.e., "2D estimate", "Single Gaussian," and "Mixture model") that have distributions similar to the test set.

**Comment by luzzou** [comment]
Got it, thanks!
